test/controllers/concerns/hal_actions/errors_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+describe HalActions::Errors do
+  describe HalActions::Errors::UnsupportedMediaType do
+    let(:subject) { HalActions::Errors::UnsupportedMediaType.new("foo") }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.